.travis.yml

```diff
@@ -32,38 +32,71 @@ jobs:
       #if: tag =~ /^release.*$/
       language: shell
       services: docker
+      install:
+        # set up awscli packages
+        - python3 -m pip install --user awscli==1.20.9
+      before_script:
+        # set up shared storage for build artifacts
+        - mkdir -p $TRAVIS_BUILD_SHARED_STORAGE
       script:
         - docker run --rm -v `pwd`:/var/task amazon/aws-sam-cli-build-image-python3.8 bash -c './build-release-package.sh'
-        - cp -p dynatrace-aws-log-forwarder.zip ~/build-stages-shared-storage/$TRAVIS_BUILD_NUMBER
+      after_success:
+        - cp -p dynatrace-aws-log-forwarder.zip $TRAVIS_BUILD_SHARED_STORAGE
+        - aws s3 sync $TRAVIS_BUILD_SHARED_STORAGE $AWS_S3_BUILD_SHARED_STORAGE
     - stage: e2e_tests
       name: Execute e2e tests
-      if: branch = master OR type = pull_request
+      # if: branch = master OR type = pull_request
+      if: branch = APM-306464

```

Comment:
hmm?

Reply:
Please see the commit message. This one is for debugging purposes only (normally e2e would run only on `master` or for a PR). I'll switch it back as soon as the build passes.